From 48c8aa7908e726f30b46fd3c5e29782f448d8f98 Mon Sep 17 00:00:00 2001
From: Harald Hope <smxi@users.noreply.github.com>
Date: Sat, 24 Mar 2018 22:08:54 -0700
Subject: [PATCH] trying manjaro id fix, missed on key thing

---
 pinxi | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/pinxi b/pinxi
index abf4455f..462686cf 100755
--- a/pinxi
+++ b/pinxi
@@ -13790,7 +13790,7 @@ sub get_linux_distro {
 	my ($distro_file) = ('');
 	my ($b_osr,@working);
 	my @derived = qw(antix-version aptosid-version kanotix-version knoppix-version 
-	mandrake-release mx-version pardus-release porteus-version sabayon-release 
+	mandrake-release manjaro-release mx-version pardus-release porteus-version sabayon-release 
 	siduction-version sidux-version slitaz-release solusos-release turbolinux-release 
 	zenwalk-version);
 	my $derived_s = join "|", @derived;
@@ -13937,7 +13937,7 @@ sub get_linux_distro {
 }
 sub get_lsb_release {
 	eval $start if $b_log;
-	my ($distro,$id,$release,$codename,$description,) = ('','','','','');
+	my ($distro,$id,$release,$codename,$description) = ('','','','','');
 	my @content = main::reader('/etc/lsb-release');
 	main::log_data('dump','@content',\@content) if $b_log;
 	@content = map {s/,|\*|\\||\"|[:\47]|^\s+|\s+$|n\/a//ig; $_} @content if @content;
@@ -13972,7 +13972,6 @@ sub get_lsb_release {
 		$distro = "$id $release $codename";
 		$distro =~ s/^\s+|\s\s+|\s+$//g; # get rid of double and trailling spaces 
 	}
-	
 	eval $end if $b_log;
 	return $distro;
 }
 
